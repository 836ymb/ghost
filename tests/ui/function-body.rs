use ghost::phantom;

fn main() {
    #[phantom]
    struct MyPhantom<T: ?Sized>;
}
