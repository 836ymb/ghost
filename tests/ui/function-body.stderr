error[E0432]: unresolved import `super::MyPhantom`
 --> tests/ui/function-body.rs:5:12
  |
5 |     struct MyPhantom<T: ?Sized>;
  |            ^^^^^^^^^ could not find `MyPhantom` in the crate root

error[E0432]: unresolved import `self`
 --> tests/ui/function-body.rs:4:5
  |
4 |     #[phantom]
  |     ^^^^^^^^^^ could not find `__value_MyPhantom` in the crate root
  |
  = note: this error originates in the attribute macro `phantom` (in Nightly builds, run with -Z macro-backtrace for more info)
